pkg/api/validation/validation.go

```diff
@@ -1438,6 +1440,11 @@ func ValidatePersistentVolume(pv *api.PersistentVolume) field.ErrorList {
 		}
 	}
 
+	if pv.Spec.VolumeType != nil && len(*pv.Spec.VolumeType) > 0 {
```

Comment:
@msau42 - ack and I listed my handle as owner, I can change if someone else feels they want it like @erinboyd or @mtanino .